apps/omg_utils/lib/omg_utils/json_rpc/response.ex

```diff
@@ -11,20 +11,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
-defmodule OMG.RPC.Web.Serializer.Response do
+defmodule OMG.Utils.HttpRPC.Response do
   @moduledoc """
   Serializes the response into expected result/data format.
 
   TODO: Intentionally we want to have single Phx app exposing both APIs, until then please keep this file similar
```

Comment:
This comment is outdated now. I think we've refactored to 2 separate WebApis with common Utils.
I'm thinking about renaming `OMG.Utils` to `OMG.HttpRPC` and find a better name for `OMG.RPC`